application/account-management/Api/Api.csproj

```diff
@@ -36,16 +40,14 @@
         <None Remove="publish\**"/>
     </ItemGroup>
 
-    <Target Name="CreateSwaggerJson" AfterTargets="Build">
-        <Exec Command="dotnet tool restore"/>
-        <Exec Command="echo true > $(OutputPath)SwaggerGenerator.tmp"/>
-        <Exec Command="dotnet swagger tofile --output $(OutputPath)..\..\..\swagger.json $(OutputPath)$(AssemblyName).dll v1"/>
-        <Delete Files="$(OutputPath)SwaggerGenerator.tmp"/>
-        <OnError ExecuteTargets="CreatingSwaggerJson"/>
-    </Target>
-
-    <Target Name="CleanupSwaggerGeneratorTmp">
-        <Delete Files="$(OutputPath)SwaggerGenerator.tmp"/>
+    <PropertyGroup>
+         <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
+     </PropertyGroup>
+
+    <Target Name="NSwag" AfterTargets="PostBuildEvent" Condition=" '$(Configuration)' == 'Debug' ">
```

Comment:
Why is this only for `Debug` builds. Is it not important that the `swagger.json` is also generated server side. Else we risk that it does not reflect the last-minute changes.

Reply:
It's accordingly to the [NSwag documentation](https://github.com/RicoSuter/NSwag/wiki/NSwag.MSBuild) - Could be a typo - I'll remove it, only reason could be related to the build information in the assembly but let's see :)